libs/feature/map/src/lib/add-layer-from-ogc-api/add-layer-from-ogc-api.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core'
+import { OgcApiEndpoint } from '@camptocamp/ogc-client'
+import { Subject, debounceTime } from 'rxjs'
+import { MapFacade } from '../+state/map.facade'
+import {
+  MapContextLayerModel,
+  MapContextLayerTypeEnum,
+} from '../map-context/map-context.model'
+
+@Component({
+  selector: 'gn-ui-add-layer-from-ogc-api',
+  templateUrl: './add-layer-from-ogc-api.component.html',
+  styleUrls: ['./add-layer-from-ogc-api.component.css'],
```

Comment:
Do you think we could make this a standalone component? that would push things towards the right direction :)

Reply:
Done !!